app/src/renderer/components/staking/TableValidators.vue

```diff
@@ -30,6 +30,7 @@ export default {
     PanelSort
   },
   props: {
+    // TODO do we really need to make this a prop? the component takes everything else from the store
```

Comment:
probably not, we should just use the validators getter - right?

Reply:
actually the validators are a filtered list, see the my delegations tab